gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -227,8 +228,11 @@ def run_compute_info(mt: hl.MatrixTable) -> hl.Table:
     ).rows()
 
     info_ht = info_ht.annotate(set_long_AS_missing_info=ht[info_ht.key])
+    info_ht = info_ht.checkpoint(
+        hl.utils.new_temp_file("compute_info_AS_missing"), overwrite=True
```

Comment:
```suggestion
        hl.utils.new_temp_file("compute_info_AS_missing", "ht"), overwrite=True
```
? (not sure if required)